### ISSUE TITLE
Crash with 1.14.4

### ISSUE THREAD

**Issue by Jipok** [shedaniel/LightOverlay-Fabric#44]
```
Mods:
  [✔️] LightOverlay-3.6.2(1)
  [✔️] OptiFine_1.14.4_HD_U_F5
  [✔️] fabric-api-0.13.1+build.257-1.14
  [✔️] optifabric-0.7.1
```
```
Java Arguments:
[-Xms512m, -Xmx9024m, -Duser.language=en]


Wrapper command is:
prime-run
```
<details>
  <summary>log</summary>

```haskell
[21:08:05] [main/INFO]: Loading for game Minecraft 1.14.4
[21:08:05] [main/WARN]: Mod `lightoverlay` (3.6.1) uses 'requires' key in fabric.mod.json, which is not supported - use 'depends'
[21:08:05] [main/WARN]: Mod `cloth` (0.6.0) uses 'requires' key in fabric.mod.json, which is not supported - use 'depends'
[21:08:05] [main/INFO]: [FabricLoader] Loading 41 mods: minecraft@1.14.4, fabric-renderer-api-v1@0.1.1+933c24c242, fabric-networking-blockentity-v0@0.2.1+28f8190f42, fabric-keybindings-v0@0.2.0+85811ffb42, fabric-command-api-v1@1.0.1+b534aa0c42, fabricloader@0.8.8+build.202, org_zeroturnaround_zt-zip@1.13, fabric-renderer-indigo@0.1.14+e15ff88642, fabric-containers-v0@0.1.3+28f8190f42, fabric-object-builder-api-v1@1.0.0+002fcb6542, fabric-dimensions-v1@0.1.1+e15ff88642, fabric-biomes-v1@0.1.1+28f8190f42, org_slf4j_slf4j-simple@1.7.29, fabric-events-interaction-v0@0.3.0+6e76156542, fabric-crash-report-info-v1@0.1.2+28f8190f42, fabric-api-base@0.1.2+28f8190f42, fabric-rendering-v0@0.1.2+28f8190f42, fabric-key-binding-api-v1@1.0.0+9e24fbc342, cloth-config2@1.1.2, fabric-rendering-data-attachment-v1@0.1.2+28f8190f42, mm@1.8, fabric-resource-loader-v0@0.1.4+28f8190f42, fabric-textures-v0@0.1.4+933c24c242, fabric-content-registries-v0@0.1.2+28f8190f42, fabric-tag-extensions-v0@0.1.3+d423005742, optifabric@0.7.1, fabric-rendering-fluids-v1@0.1.3+28f8190f42, fabric-registry-sync-v0@0.2.3+28f8190f42, fabric-commands-v0@0.2.0+bfd4c99d42, lightoverlay@3.6.1, fabric-mining-levels-v0@0.1.1+28f8190f42, fabric-loot-tables-v1@0.1.1+28f8190f42, fabric-events-lifecycle-v0@0.1.1+f6e5d2dc42, fabric@0.13.1+build.257-1.14, org_slf4j_slf4j-api@1.7.29, fabric-particles-v1@0.2.1+e9056b2242, fabric-models-v0@0.1.0+933c24c242, fabric-item-groups-v0@0.2.0+7ce6c57742, fabric-object-builders-v0@0.2.0+002fcb6542, fabric-networking-v0@0.1.10+e7757afe42, cloth@0.6.0
[21:08:05] [main/INFO]: SpongePowered MIXIN Subsystem Version=0.8 Source=file:/home/kiv/.multimc/libraries/net/fabricmc/sponge-mixin/0.8+build.18/sponge-mixin-0.8+build.18.jar Service=Knot/Fabric Env=CLIENT
Found existing patched optifine jar, using that
[21:08:06] [main/INFO]: [OptiFine] *** Reflector Forge ***
[21:08:06] [main/INFO]: [OptiFine] (Reflector) Class not present: mods.betterfoliage.client.BetterFoliageClient
[21:08:06] [main/INFO]: [OptiFine] (Reflector) Method not present: net.minecraft.class_773.getTexture
Replacement `renderBlock` call:  method_3355.(Lnet/minecraft/class_2680;Lnet/minecraft/class_2338;Lnet/minecraft/class_1920;Lnet/minecraft/class_287;Ljava/util/Random;)Z
[21:08:06] [main/INFO]: [OptiFine] (Reflector) Method not present: net.minecraft.class_842.getModelData
[21:08:07] [main/INFO]: [OptiFine] (Reflector) Method not present: net.minecraft.class_1299.getTags
[21:08:07] [main/INFO]: [OptiFine] (Reflector) Field not present: net.minecraft.class_1309.NAMETAG_DISTANCE
[21:08:07] [main/INFO]: [OptiFine] (Reflector) Method not present: net.minecraft.class_3611.getTags
[21:08:07] [main/INFO]: [OptiFine] (Reflector) Method not present: net.minecraft.class_304.setKeyConflictContext
[21:08:07] [main/INFO]: [OptiFine] (Reflector) Method not present: net.minecraft.class_304.setKeyModifierAndCode
[21:08:07] [main/INFO]: [OptiFine] (Reflector) Method not present: net.minecraft.class_304.getKeyModifier
[21:08:07] [main/INFO]: [OptiFine] (Reflector) Method not present: net.minecraft.class_790.fromJson
[21:08:07] [main/INFO]: [OptiFine] (Reflector) Method not present: net.minecraft.class_296$class_298.preDraw
[21:08:07] [main/INFO]: [OptiFine] (Reflector) Method not present: net.minecraft.class_296$class_298.postDraw
[21:08:07] [main/INFO]: [OptiFine] (Reflector) Field not found: World.tileEntitiesToBeRemoved
[21:08:07] [main/INFO]: [OptiFine] *** Reflector Vanilla ***
[21:08:07] [main/INFO]: Setting user: Player
[21:08:10] [main/INFO]: Definitely not up to no good
[21:08:11] [main/INFO]: [Indigo] Registering Indigo renderer!
[21:08:11] [main/INFO]: [Indigo] Compatibility mode enabled.
---- Minecraft Crash Report ----
// Quite honestly, I wouldn't worry myself about that.

Time: 6/23/20 9:08 PM
Description: Initializing game

java.lang.RuntimeException: Could not execute entrypoint stage 'client' due to errors, provided by 'lightoverlay'!
	at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke0(EntrypointUtils.java:53)
	at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke(EntrypointUtils.java:36)
	at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointClient.start(EntrypointClient.java:33)
	at net.minecraft.class_310.method_1503(class_310.java)
	at net.minecraft.class_310.method_1514(class_310.java:394)
	at net.minecraft.client.main.Main.main(Main.java:155)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:192)
	at net.fabricmc.loader.launch.knot.Knot.init(Knot.java:140)
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:26)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:196)
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:231)
	at org.multimc.EntryPoint.listen(EntryPoint.java:143)
	at org.multimc.EntryPoint.main(EntryPoint.java:34)
Caused by: java.lang.NoSuchFieldError: INSTANCE
	at me.shedaniel.lightoverlay.fabric.LightOverlay.onInitializeClient(LightOverlay.java:118)
	at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke0(EntrypointUtils.java:50)
	... 20 more


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Client thread
Stacktrace:
	at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke0(EntrypointUtils.java:53)
	at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointUtils.invoke(EntrypointUtils.java:36)
	at net.fabricmc.loader.entrypoint.minecraft.hooks.EntrypointClient.start(EntrypointClient.java:33)
	at net.minecraft.class_310.method_1503(class_310.java)

-- Initialization --
Details:
Stacktrace:
	at net.minecraft.class_310.method_1514(class_310.java:394)
	at net.minecraft.client.main.Main.main(Main.java:155)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:192)
	at net.fabricmc.loader.launch.knot.Knot.init(Knot.java:140)
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:26)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:196)
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:231)
	at org.multimc.EntryPoint.listen(EntryPoint.java:143)
	at org.multimc.EntryPoint.main(EntryPoint.java:34)

-- System Details --
Details:
	Minecraft Version: 1.14.4
	Minecraft Version ID: 1.14.4
	Operating System: Linux (amd64) version 5.4.44_1
	Java Version: 1.8.0_252, Void
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Void
	Memory: 1102742728 bytes (1051 MB) / 1429733376 bytes (1363 MB) up to 8411152384 bytes (8021 MB)
	CPUs: 8
	JVM Flags: 2 total; -Xms512m -Xmx9024m
	Fabric Mods: 
		cloth: Cloth Events 0.6.0
		cloth-config2: Cloth Config v2 1.1.2
		fabric: Fabric API 0.13.1+build.257-1.14
		fabric-api-base: Fabric API Base 0.1.2+28f8190f42
		fabric-biomes-v1: Fabric Biomes (v1) 0.1.1+28f8190f42
		fabric-command-api-v1: Fabric Command API (v1) 1.0.1+b534aa0c42
		fabric-commands-v0: Fabric Commands (v0) 0.2.0+bfd4c99d42
		fabric-containers-v0: Fabric Containers (v0) 0.1.3+28f8190f42
		fabric-content-registries-v0: Fabric Content Registries (v0) 0.1.2+28f8190f42
		fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.1.2+28f8190f42
		fabric-dimensions-v1: fabric-dimensions-v1 0.1.1+e15ff88642
		fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.3.0+6e76156542
		fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.1.1+f6e5d2dc42
		fabric-item-groups-v0: Fabric Item Groups (v0) 0.2.0+7ce6c57742
		fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.0+9e24fbc342
		fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.0+85811ffb42
		fabric-loot-tables-v1: Fabric Loot Tables (v1) 0.1.1+28f8190f42
		fabric-mining-levels-v0: Fabric Mining Levels (v0) 0.1.1+28f8190f42
		fabric-models-v0: Fabric Models (v0) 0.1.0+933c24c242
		fabric-networking-blockentity-v0: Fabric Networking Block Entity (v0) 0.2.1+28f8190f42
		fabric-networking-v0: Fabric Networking (v0) 0.1.10+e7757afe42
		fabric-object-builder-api-v1: Fabric Object Builder API (v1) 1.0.0+002fcb6542
		fabric-object-builders-v0: Fabric Object Builders (v0) 0.2.0+002fcb6542
		fabric-particles-v1: fabric-particles-v1 0.2.1+e9056b2242
		fabric-registry-sync-v0: Fabric Registry Sync (v0) 0.2.3+28f8190f42
		fabric-renderer-api-v1: Fabric Renderer API (v1) 0.1.1+933c24c242
		fabric-renderer-indigo: Fabric Renderer - Indigo 0.1.14+e15ff88642
		fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.1.2+28f8190f42
		fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 0.1.3+28f8190f42
		fabric-rendering-v0: Fabric Rendering (v0) 0.1.2+28f8190f42
		fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.1.4+28f8190f42
		fabric-tag-extensions-v0: Fabric Tag Extensions (v0) 0.1.3+d423005742
		fabric-textures-v0: Fabric Textures (v0) 0.1.4+933c24c242
		fabricloader: Fabric Loader 0.8.8+build.202
		lightoverlay: Light Overlay 3.6.1
		minecraft: Minecraft 1.14.4
		mm: Manningham Mills 1.8
		optifabric: Optifabric 0.7.1
		org_slf4j_slf4j-api: slf4j-api 1.7.29
		org_slf4j_slf4j-simple: slf4j-simple 1.7.29
		org_zeroturnaround_zt-zip: zt-zip 1.13
	Launched Version: MultiMC5
	LWJGL: 3.2.2 build 10
	OpenGL: NO CONTEXT
	GL Caps: 
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fabric'
	Type: Client (map_client.txt)
	Resource Packs: ~~ERROR~~ NullPointerException: null
	Current Language: ~~ERROR~~ NullPointerException: null
	CPU: <unknown>
	OptiFine Version: OptiFine_1.14.4_HD_U_F5
	OptiFine Build: 20191204-141250
	Shaders: null
	OpenGlVersion: null
	OpenGlRenderer: null
	OpenGlVendor: null
	CpuCount: 0
```
</details>

**Comment by shedaniel** [comment]
Will fix.

**Comment by shedaniel** [comment]
Fixed in v3.7.0
